Makefile

```diff
@@ -47,6 +48,13 @@ ifeq (aarch64, $(TARGET))
   DASM_ARCH  = aarch64
   DASM_FLAGS = -M
 endif
+ifeq (freebsd, $(OS))
+  CC=clang
+  BUILD_CC=clang
+  override CFLAGS += -I/usr/local/include
+  override BUILD_CFLAGS += -I/usr/local/include
+  LDFLAGS += -L/usr/local/lib -lutil
```

Comment:
why do you need ``/usr/local/include``?

Reply:
Do you know what symbols are imported from  ``util``?

Reply:
> why do you need `/usr/local/include`?

capstone

Reply:
> Do you know what symbols are imported from `util`?

kinfo_getproc

Reply:
Can we avoid this dependency using ``sysctl``? Like in example at https://man.freebsd.org/cgi/man.cgi?query=sysctl&sektion=3&apropos=0&manpath=FreeBSD+10.0-RELEASE

Reply:
> capstone (in FreeBSD non system packages are in /usr/local)

Did you build capstone yourself?

Reply:
Not at all, FreeBSD package system put all in /usr/local

Reply:
> https://man.freebsd.org/cgi/man.cgi?query=sysctl&sektion=3&apropos=0&manpath=FreeBSD+10.0-RELEASE

should be possible I ll have a look later in the day